grails-app/controllers/org/pih/warehouse/api/CycleCountApiController.groovy

```diff
@@ -38,6 +39,12 @@ class CycleCountApiController {
         render([data: cycleCounts, totalCount: cycleCounts.totalCount] as JSON)
     }
 
+    def getCandidatesInProgress(CycleCountCandidateFilterCommand filterParams) {
+        List<CycleCountCandidateInProgress> tabRecords = cycleCountService.getCandidatesInProgress(filterParams, params.facilityId)
+
+        render([data: tabRecords, totalCount: tabRecords.totalCount] as JSON)
```

Comment:
don't you also need to support exporting as csv like we have with the above method?

Reply:
I forgot to mention that I left that for now, to push the initial draft. Moreover - the csv returns last counted date, which Justin said probably said caused the most issues, and as long as we don't need it, we are fine, so we can still use the old endpoint for the csv for now.